### DISCUSSION TITLE
Git clone error while initiating renovate scan

### DISCUSSION THREAD

**Discussion by nagarajui7** [renovatebot/renovate#11731] [Request Help]
Hi Team,

I ran the renovate scan on a repo, but facing the below mentioned error:
```fatal: unable to checkout working tree\nwarning: Clone succeeded, but checkout failed.\nYou can inspect what was checked out with 'git status'\nand retry with 'git restore --source=HEAD```

Can you guys help me on this ?

**Comment by rarkins** [comment]
Try https://www.google.com/search?q=Clone+succeeded%2C+but+checkout+failed
